contracts/campaignAirdrops/NewERC1155DropCampaign.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "../interfaces/AirdropMerkleProof.sol";
+import "../interfaces/IAirBroFactory.sol";
+
+/// @title Airdrops existing ERC1155 tokens for airdrop recipients
+contract NewERC1155DropCampaign is ERC1155, AirdropMerkleProof {
+    address public immutable airbroCampaignFactoryAddress;
+    uint8 private immutable _tokenId = 0; // yes, for immutable/constant I had to set the variable to = 0;
```

Comment:
constant and not immutable, you want immutable when initializing in constructor

Reply:
For constants and immutables it is better to use uint256 than uint8
Feel free to copy to remix and test deployment cost
```
pragma solidity 0.8.15;

// 80231
contract Asm {    
    uint256 public constant asd = 1;
}

// 81095
contract Asm2 {    
    uint8 public constant asd = 1;
}
```

Reply:
@GotchaGuy 
Balance after Asm deployment: 99.99999999999990**8125**
Balance after Asm2 deployment: 99.99999999999990**7477**

Gas saved with Asm2: **648**